### DISCUSSION TITLE
I wonder where is the development documentation for the plugin

### DISCUSSION THREAD

**Discussion by coolf233** [GetPublii/Publii#1683] [Help] [answered]
It is very difficult to visit the China region of most of the comments plug-in, I want to know about the development manual of the plug-in, but I did not find the relevant documentation in the developer manual, can you provide it?

**Comment by bobmitro** [comment, marked as answer]
Just look at this: https://github.com/GetPublii/Publii/discussions/1359

**Reply by coolf233** [reply]
Thank you. Maybe this is just what I need

**Comment by candidexmedia** [comment]
Are you looking for the docs for a particular commenting platform's plugin? If so, please specify which one.

**Reply by coolf233** [reply]
No, I'm looking for publii plugin development documentation
